src/flanders/utils.cljc

```diff
@@ -58,12 +58,18 @@
   (loop [ddl-loc (->ddl-zip ddl)]
     (cond
       (z/end? ddl-loc) (z/root ddl-loc)
-      (fp/entry ddl-loc) (recur
-                          (z/next
-                           (z/replace ddl-loc
-                                      (assoc (z/node ddl-loc)
-                                             :required? required?))))
-      :else (recur (z/next ddl-loc)))))
+      (fp/entry ddl-loc)
+      (recur
+       (z/next
+        (if (get-in (z/node ddl-loc) [:key :open?])
+          (z/replace ddl-loc
+                     (assoc (z/node ddl-loc)
+                            :required? true))
+          (z/replace ddl-loc
+                     (assoc (z/node ddl-loc)
+                            :required? required?)))))
```

Comment:
I think open vocab can be optional. Your issue seems to be related to the [`schema.clj`](https://github.com/threatgrid/flanders/blob/master/src/flanders/schema.cljc#L54) generator.